### DISCUSSION TITLE
createMultibody cannot get link state and joint state

### DISCUSSION THREAD

**Discussion by zhangzhongwd** [bulletphysics/bullet3#3683] [General]
Because I need to learn mechanical configuration using by reinforce learning, I need to use createMultibody() instead loadurdf directly. But I found when I used createMultibody to get objectUid, I cannot get linkstates or get jointstates. getlinkstates() will return NULL. In addition, getjointstates() return wrong values and remain static.

**Comment by unishuai** [comment]
hello,I want to get link state now,but it can still not work perfectly。When  using createMultibody to create many links,i can just get first link state
